pkg/minikube/cluster/ip.go

```diff
@@ -65,15 +65,13 @@ func HostIP(host *host.Host, clusterName string) (net.IP, error) {
 		if err != nil {
 			return []byte{}, errors.Wrap(err, "Error getting IP address")
 		}
-		// user network case
 		if ipString == "127.0.0.1" {
+			// user network case
```

Comment:
change this to:
// qemu driver: user network case

Reply:
This is already in `case driver.QEMU:` though
```
	case driver.QEMU2:
		ipString, err := host.Driver.GetIP()
		if err != nil {
			return []byte{}, errors.Wrap(err, "Error getting IP address")
		}
		if ipString == "127.0.0.1" {
			// user network case
			return net.ParseIP("10.0.2.2"), nil
		} else {
			// socket_vmnet network case
			return net.ParseIP("192.168.105.1"), nil
		}
```

Reply:
fair enough !